feature-utils/poly-import/src/entities/data-account.js

```diff
@@ -0,0 +1,57 @@
+export default class DataAccount {
+    constructor() {
+        this._importingResults = [];
+        this._importedFileNames = [];
+
+        this._processedData = [];
+        this._name = "";
```

Comment:
since this is possibly affecting the way that the dataAccount object is put together in the importers execution, it is probably better to change this in another PR as to keep the files affected by this PR low.